File: MODUL 4/SOAL 2/PRAK402-2410817210026-Fadhil_Syahdama_Mahatma_Putra.c
#include <stdio.h>

int main() {
    int botan, i;

    scanf("%d", &botan);

    for(i = 1; i <= botan; i += 2) {
        printf("%d ", i);
    }

    printf("\n");

    if (botan % 2 == 0) { 
        for (i = botan; i >= 2; i -= 2) {
            printf("%d ", i);
        }
    /*} else { 
        for (i = botan - 1; i >= 2; i -= 2) {
            printf("%d ", i);
        }
    }*/
}
}

File: MODUL 4/SOAL 2/PRAK402-2410817210026-Fadhil_Syahdama_Mahatma_Putra.py
botan = int(input())

for i in range(1, botan + 1, 2):
    print(i, end=' ')

print()

if botan % 2 == 0:
    for i in range(botan, 1, -2):
        print(i, end=' ')
else:
    for i in range(botan - 1, 1, -2):
        print(i, end=' ')